src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj

```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataLake.Analytics</PackageId>
     <Description>Provides Data Lake Analytics account, job and catalog management capabilities for Microsoft Azure.</Description>
-    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionPrefix>3.0.1</VersionPrefix>
     <AssemblyName>Microsoft.Azure.Management.DataLake.Analytics</AssemblyName>
     <PackageTags>Microsoft Azure Data Lake Analytics management;DataLakeAnalytics;Data Lake Analytics management;REST HTTP client;windowsazureofficial;netcore451511</PackageTags>
-    <PackageReleaseNotes>See https://github.com/azure/azure-sdk-for-net/blob/vs17Dev/src/SDKs/DataLake.Analytics/changelog.md for release notes.</PackageReleaseNotes>
+    <PackageReleaseNotes>See <href>https://aka.ms/adladotnetsdkchangelog</href> for release notes.</PackageReleaseNotes>
```

Comment:
@begoldsm please test out this change by locally creating a nuget, adding it to localFeed and see the UI in VS when it tries to install the nuget package, it will present you an agreement UI that will have the ReleaseNotes in the VS UI.
You can create a local nuget package by running msbuild build.proj /t:CreateNugetPackage /p:Scope=SDKs\DataLake.Analytics
Please do this testing and then introduce this change.